### PR TITLE
[FTL-1705] Create wallet-node-sdk

### PR DESCRIPTION
**Pull request by inga-affinidi**
A quick fix for FTL-1705, wallet-node-sdk is the exact copy of wallet-browser-sdk in this version.
(Will be refactored later in https://github.com/affinityproject/affinidi-core-sdk/pull/108, among with everything else)

### CODE REVIEW

**Review by vitalyrudenko-affinidi — APPROVED** [review]
lgtm

File: sdk/node/src/AffinityWallet.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { EventComponent } from '@affinidi/affinity-metrics-lib'
+import { CommonNetworkMember as CoreNetwork, __dangerous } from '@affinidi/wallet-core-sdk'
+
+import platformEncryptionTools from './PlatformEncryptionTools'
+
+export type SdkOptions = __dangerous.SdkOptions & {
+  issueSignupCredential?: boolean
+}
+
+const COMPONENT = EventComponent.AffinidiBrowserSDK
```
Comment (Yengas) [inline sdk/node/src/AffinityWallet.ts:10]: I think this is the enum that is used when sending SDK metrics. Did we miss creating a new enum and using it here?

Comment (inga-affinidi) [reply sdk/node/src/AffinityWallet.ts:10]: This was intended to be a quick fix to unblock work on BBS task. You're right, this enum should get a new value, and I'm going to do this after I'll return from vacation

Comment (inga-affinidi) [reply sdk/node/src/AffinityWallet.ts:10]: (The problem here is that EventComponent is defined in affinity-metrics-lib which is another project and package entirely)

**Review by Yengas — DISMISSED** [review]

File: sdk/node/src/AffinityWallet.ts (hunk 1 above)
Comment (Yengas) [reply sdk/node/src/AffinityWallet.ts:10]: Please don't forget to take a note / create task for it.

**Review by vitalyrudenko-affinidi — APPROVED** [review]

**Review by Yengas — APPROVED** [review]

**Pull request merged**